common/src/main/resources/less/gpp-grid.less

```diff
@@ -1,6 +1,15 @@
 @two-column-break: 60em;
 @three-column-break: 90em;
 
+/*
+ Overrides the default values of fields inside an SUI form.
+ These values interfere with the gpp-grid layout.
+ */
+.ui.form.gpp-grid .field {
+    margin: 0;
```

Comment:
I was following prior art, but should we change all the `gpp` to `explore` or `lucuma`, and rename `GPPStyles` similarly?

Reply:
I suppose we should, maybe explore considering it is explore specific but if it is going to be used more broadly, lucuma?